projects/plugins/jetpack/extensions/blocks/markdown/edit.js

```diff
@@ -1,9 +1,9 @@
 import { BlockControls, PlainText } from '@wordpress/block-editor';
 import { compose } from '@wordpress/compose';
 import { withDispatch, withSelect } from '@wordpress/data';
-import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import classnames from 'classnames';
+import { useEffect, useRef, useState } from 'react';
```

Comment:
Something I wanted to note / ask, since I've seen you do that in the past few PRs.

What would you think about using the versions from the `@wordpress/element` package instead? I know we kinda of use both in our blocks, but I'm thinking it may be nicer to be consistent and rely on WordPress packages whenever possible.

Reply:
I'll take not to update those in the next sprint.